app/utils/shop.py

```diff
@@ -13,13 +14,24 @@ def price(self, amount=1):
         return {item: self.prices[item] * amount for item in self.prices}
 
 
+
 class UpgradableItem:
-    def __init__(self, level_1: ShopItem, level_2: ShopItem, level_3: ShopItem, level_4: ShopItem, level_5: ShopItem):
+    def __init__(self, level_1: ShopItem=None, level_2: ShopItem=None,
```

Comment:
do:
```suggestion
    def __init__(self, level_1: ShopItem=None, level_2: ShopItem=None,
    def __init__(self,
                 ShopItem = None, level_2: ShopItem = None,
```
And remember spaces on the sides of the `=`

Reply:
alright

Reply:
danke bitch